engine/src/test/java/net/jqwik/engine/properties/shrinking/UnshrinkableTests.java

```diff
@@ -0,0 +1,33 @@
+package net.jqwik.engine.properties.shrinking;
+
+import net.jqwik.api.Arbitraries;
+import net.jqwik.api.Example;
```

Comment:
jqwik coding style is to use only wildcard imports, e.g. `import net.jqwik.api.*`